posthog-android/src/main/java/com/posthog/android/replay/internal/Debouncer.kt

```diff
@@ -31,6 +45,6 @@ internal class Debouncer(
     }
 
     companion object {
-        private const val ONE_FRAME_MS = 64L
+        private const val DELAY_MS = 500L
```

Comment:
I can almost see us letting someone configure this... not-blocking here just seeding the idea in your head :)

Reply:
yeah, I thought the same, lets see if this approach works better first, and happy to make it configurable otherwise.